### ISSUE TITLE
Adding CSS properties using shorthand

### ISSUE THREAD

**Issue by fakdora** [styled-system/styled-system#1424]
**Adding CSS properties.**

Have a question about this from the doc
> Shortcut definition
> If your style prop does not need to pick up values from the theme and the prop matches the intended CSS property, you can use a shortcut definition in the system function argument.
> 
> system({
>   transition: true,
> })
> This will take the transition prop and translate it into a style object.

So, if I want to add CSS properties that doesn't exist in styled-system's API such as transistion, whiteSpace, textIndent, textDecoration, etc, do I just declare it like below

```
system ({
  textDecoration: true,
  textIndent: true,
  textTransform: true,
  transform: true,
  transition: true,
  whiteSpace: true,
});
```

and they will be accepted, right? 
Is it safe to assume that it translates CamelCase properties to the corresponding CSS in camel-case hyphenation format?

thank you

**Comment by ivanquirino** [comment]
@fakdora That's correct. Those properties you specified in your example will be converted as is to CSS. Just follow the docs and you'll have no problem.
